providers/hubspot/subscriptionEvent.go

```diff
@@ -9,24 +9,40 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/amp-labs/connectors/common"
 )
 
+/*
 type SubscriptionEvent struct {
 	AppId            int    `json:"appId"`
 	EventId          int    `json:"eventId"`
 	SubscriptionId   int    `json:"subscriptionId"`
 	PortalId         int    `json:"portalId"`
-	OccurredAt       int    `json:"occurredAt"`
+	OccurredAt       int    `json:"occurredAt"` // in milliseconds
 	SubscriptionType string `json:"subscriptionType"`
 	AttemptNumber    int    `json:"attemptNumber"`
-	ObjectId         int    `json:"objectId"`
 	ChangeSource     string `json:"changeSource"`
-	PropertyName     string `json:"propertyName"`
-	PropertyValue    string `json:"propertyValue"`
+	// Optional fields
+	ObjectId   *int    `json:"objectId,omitempty"`
+	ChangeFlag *string `json:"changeFlag,omitempty"`
+	// Property Change Fields
+	PropertyName  *string `json:"propertyName,omitempty"`
+	PropertyValue *string `json:"propertyValue,omitempty"`
+	// Association Change Fields
+	AssociationType      *string `json:"associationType,omitempty"`
+	FromObjectId         *int    `json:"fromObjectId,omitempty"`
+	ToObjectId           *int    `json:"toObjectId,omitempty"`
+	AssociationRemoved   *bool   `json:"associationRemoved,omitempty"`
+	IsPrimaryAssociation *bool   `json:"isPrimaryAssociation,omitempty"`
 }
 
+
+*/
+
+type SubscriptionEvent map[string]any
```

Comment:
As discussed, going with a map is a good approach in order to preserve the full raw event body to send to builders (whereas JSON marshaling to/from a struct will lose the fields that are not captured in the struct definition). Can you please add a comment to explain this?